newrelic/alert/alert_policy.tf

```diff
@@ -0,0 +1,14 @@
+resource "newrelic_alert_policy" "tfer--new-002D-host-002D-usage-002D-against-002D-license-002D-695505" {
+  incident_preference = "PER_POLICY"
+  name                = "New Host Usage Against License"
+}
+
+resource "newrelic_alert_policy" "tfer--outage-002D-712624" {
+  incident_preference = "PER_POLICY"
+  name                = "Outage"
+}
+
+resource "newrelic_alert_policy" "tfer--usage-002D-code-002D-gov-002D-synthetics-002D-695765" {
+  incident_preference = "PER_POLICY"
+  name                = "Usage code.gov Synthetics "
```

Comment:
The idea is to have a bunch of these, right? Totally fine if we are taking care of it separately, but just wanted to check.

Reply:
I have to manually create these in the UI... these are just the id's that match those policies and ensure they aren't removed once created in the UI